src/components/TabletApp/RightSide/Menu/Menu.tsx

```diff
@@ -0,0 +1,92 @@
+import { Button, Box, Typography, Skeleton } from "@mui/material";
+import React from "react";
+import COLORS from "../../../../constants/CustomColors";
+
+import QuickBook from "../QuickBook/QuickBook";
+import { Link } from "react-router-dom";
+import CalendarMonthIcon from "@mui/icons-material/CalendarMonth";
```

Comment:
Maybe change the order of the imports to separate the react ones from our imports